backend/Services/ISightingsService.cs

```diff
@@ -0,0 +1,8 @@
+using WhaleSpotting.Models.Request;
+
+namespace WhaleSpotting.Services;
+
+public interface ISightingsService
```

Comment:
I might be missing something, what's the benefit of having this interface rather than directly using the service?

Reply:
Moved ISightingsService into SightingsService.cs